### DISCUSSION TITLE
Como baixar versao anterior?

### DISCUSSION THREAD

**Discussion by laura-svarta** [arkypita/LaserGRBL#2261] [Q&A]
O programa atualizou e a versão nova não esta funcionando direito pra mim, estragou meu trabalho! Quero voltar para a versão anterior. Alguem pode me ajudar??

**Comment by arkypita** [comment]
It is always possible to go back to a previous version, however I invite you to give me some more information and do some tests to try to understand what the problem is. Keep in mind that the problem you had may not depend on the new version but on an external problem, so it would be important to do more tests and try to understand the cause.

1. Can you better describe the type of problem you had?
2. Can you try deactivating the new dynamic preview window (menu "grbl", "settings", "options", "legacy preview") and see if the problem goes away?

If you want to do some tests without having to ruin the materials you can always disconnect the cable that powers the laser, in this way the machine will carry out the work without engraving.

The list of all versions is [here](https://github.com/arkypita/LaserGRBL/releases), the previous stable version is [6.2.2](https://github.com/arkypita/LaserGRBL/releases/tag/v6.2.2)
